core/templates/dev/head/components/forms/schema_editors/schema_based_html_editor_directive.html

```diff
@@ -1,4 +1,10 @@
-<text-angular-rte ng-if="!isDisabled()" html-content="$parent.localValue"
+<!-- If the current RTE is CKEditor then render CKEditor as the RTE. -->
+<ck-editor-rte ng-if="!isDisabled() && isCurrentRTECKEditor()" ng-model="$parent.localValue"
+               ui-config="uiConfig()" label-for-focus-target="labelForFocusTarget()">
```

Comment:
Note that having a labelForFocusTarget attr in the directive is inconsistent with not passing it in. Either reinstate it there or remove it here.

Please also document the rationale for removal since this deviates from what we currently do with textAngular.

Reply:
Done!

Also from what I found:

- CKEditor has a focusManager to manage the focus activity for the currently active instance, so this feature is not required for CKEditor. (https://docs-old.ckeditor.com/ckeditor_api/symbols/CKEDITOR.focusManager.html)
- In TextAngular, I did not find any service for managing focus and focus needs to be set pragmatically. (https://github.com/textAngular/textAngular/issues/687)

PTAL @AllanYangZhou  @seanlip